### PR TITLE
Revert CRDs move to v1, move back to v1beta1

### PR DESCRIPTION
**Pull request by nikolay-pshenichny**
This change partially reverts #1495. It is the second step of a series of PRs that will move back to using v1beta1 APIs to add back support for kubernetes 1.15.

First PR is here - https://github.com/istio/tools/pull/1138

@joeyb , @ramaraochavali

### CODE REVIEW

File: common/scripts/setup_env.sh [hunk 1]
```diff
@@ -59,7 +59,9 @@ fi
 
 # Build image to use
 if [[ "${IMAGE_VERSION:-}" == "" ]]; then
-  export IMAGE_VERSION=master-2020-08-12T13-43-22
+  # TODO: Update with the new image when https://github.com/istio/tools/pull/1138 is merged
```
Comment (nikolay-pshenichny) [inline common/scripts/setup_env.sh:62]: NOTE: This PR needs to be updated with a new version of cue-gen when https://github.com/istio/tools/pull/1138 is merged.

**Comment by nikolay-pshenichny** [comment]
@rshriram , could you please review/approve the 1st PR here - https://github.com/istio/tools/pull/1138.
When it is merged, I will update this one to use the latest `cue-gen`.
Thank you!

File: common/scripts/setup_env.sh [hunk 2]
```diff
@@ -59,7 +59,7 @@ fi
 
 # Build image to use
 if [[ "${IMAGE_VERSION:-}" == "" ]]; then
-  export IMAGE_VERSION=master-2020-08-12T13-43-22
+  IMAGE_VERSION=master-2020-08-19T17-29-34
```
Comment (nikolay-pshenichny) [inline common/scripts/setup_env.sh:62]: latest image from https://console.cloud.google.com/gcr/images/istio-testing/GLOBAL/build-tools?gcrImageListsize=30

Comment (howardjohn) [reply common/scripts/setup_env.sh:62]: @nikolay-pshenichny the flow here is pretty complex, but basically once the image is built you should modify this file in istio/common-files (same file as here - its a mirror). That will then get used by the CI job to start using the image.

When you change it here its only impacting the image local devlopers will use, unfortunately the CI does not read this and is configured seperately.

tl;dr update istio/common-files then we can merge this

**Comment by nikolay-pshenichny** [comment]
@howardjohn, this is the second PR in the series.

File: common/scripts/setup_env.sh (hunk 2 above)
Comment (nikolay-pshenichny) [reply common/scripts/setup_env.sh]: Thank you, @howardjohn ! 
I submitted a PR here - https://github.com/istio/common-files/pull/301

**Comment by howardjohn** [comment]
/retest

should be updated now
